### DISCUSSION TITLE
Scan US Driver's License PDF417

### DISCUSSION THREAD

**Discussion by NagRamachandran** [mebjas/html5-qrcode#280] [Q&A]
Hi, I am unable to scan driver's license PDF417 from Windows, MAC, Android and IOS devices using camera. Is there any specific settings? Thanks!

**Comment by mebjas** [comment]
Can you share an example image?

**Reply by outprove** [reply]
@mebjas  I am having a similar problem. Using the demo,  I'm not able to scan a PDF417 bar code on the back of my Virginia license (for example). It scans other codes like a champ. 

As a test I went to https://websdk.scandit.com/ to try a commercial solution and it scanned the PDF417 fine. I

Not sure if there's a trick or a setting - I tried specifying allowable types, but that didn't help. 

Thanks for maintaining such a great app!
